### PR TITLE
Add a "only-version" version scheme

### LINKED ISSUE #1002: Add a `just-the-tag` version scheme
Howdy! :wave: 

Love the tool/library. Hoping to use it for https://github.com/pantsbuild/pants versioning.

One thing is that we use `.devN` versions for the development series, so (most) of the version schemes that exist are incompatible if there's any kind of distance from the tag (the only ones that don't assert either create an illegal version like `.devN.postM` or blast the dev number).

Really, the distance is nice to know, but also isn't crucial to us, so meh.

### PR DESCRIPTION
**Pull request by thejcannon**
Fixes https://github.com/pypa/setuptools_scm/issues/1002 by adding a "only-version" version scheme.

### CODE REVIEW

File: testing/test_version.py [hunk 1]
```diff
@@ -169,6 +170,32 @@ def test_bump_dev_version_nonzero_raises() -> None:
     with pytest.raises(ValueError, match=match):
         guess_next_version(m("1.0.dev1"))
 
+@pytest.mark.parametrize(
+    "version",
+    [
+        "1.dev0",
```
Comment (thejcannon) [inline testing/test_version.py:177]: Ripped from https://peps.python.org/pep-0440/

**Review by RonnyPfannschmidt — APPROVED** [review]
I believe the name should be only-tag and the docs need a warning that this means version is no longer pseudo unique per commit

Still adding a blanket approval

I'll try to get this in this morning or by the weekend

**Comment by thejcannon** [comment]
Done and done.

**Comment by RonnyPfannschmidt** [comment]
thanks, would you like to see this squashed or should i merge as is?

**Comment by thejcannon** [comment]
Oh I'm definitely camp squash-merge. "Rename" isn't much of an inspiring commit message :joy:

**Comment by thejcannon** [comment]
@RonnyPfannschmidt when might there be a new version of `setuptools_scm` to use?
Also, if there is going to be one soon, might it include https://github.com/pypa/setuptools_scm/pull/1004?

Thanks! 🙏

**Comment by RonnyPfannschmidt** [comment]
I'll try to merge the clean up mr soon and releas this week

**Comment by jelacious** [comment]
Here's a bump on getting that release up and going as I'd like to use `only-version`. <3

**Pull request merged**